pcdsdevices/lodcm.py

```diff
@@ -186,4 +407,276 @@ def remove_dia(self, moved_cb=None, timeout=None, wait=False):
+
+        th2_user = get_status_value(status_info, 'th2', 'position')
+        th2_dial = get_status_value(status_info, 'th2', 'dial_position',
+                                    'value')
+
+        chi2_user = get_status_value(status_info, 'ch2', 'position')
+        chi2_dial = get_status_value(status_info, 'ch2', 'dial_position',
+                                     'value')
+
+        y2_user = get_status_value(status_info, 'y2', 'position')
+        y2_dial = get_status_value(status_info, 'y2', 'dial_position', 'value')
+
+        hn2_user = get_status_value(status_info, 'h2n', 'position')
+        hn2_dial = get_status_value(status_info, 'h2n', 'dial_position',
+                                    'value')
+
+        hp2_user = get_status_value(status_info, 'h2p', 'position')
+        hp2_dial = get_status_value(status_info, 'h2p', 'dial_position',
+                                    'value')
+
+        diode2_user = get_status_value(status_info, 'diode2', 'position')
+        diode2_dial = get_status_value(status_info, 'diode2', 'dial_position',
+                                       'value')
+        # diagnostics
+        dh_units = get_status_value(status_info, 'dh', 'user_setpoint',
+                                    'units')
+        dh_user = get_status_value(status_info, 'dh', 'position')
+        dh_dial = get_status_value(status_info, 'dh', 'dial_position', 'value')
+
+        dv_units = get_status_value(status_info, 'dv', 'user_setpoint',
+                                    'units')
+        dv_user = get_status_value(status_info, 'dv', 'position')
+        dv_dial = get_status_value(status_info, 'dv', 'dial_position', 'value')
+
+        dr_units = get_status_value(status_info, 'dr', 'user_setpoint',
+                                    'units')
+        dr_user = get_status_value(status_info, 'dr', 'position')
+        dr_dial = get_status_value(status_info, 'dr', 'dial_position', 'value')
+
+        df_units = get_status_value(status_info, 'df', 'user_setpoint',
+                                    'units')
+        df_user = get_status_value(status_info, 'df', 'position')
+        df_dial = get_status_value(status_info, 'df', 'dial_position', 'value')
+
+        dd_units = get_status_value(status_info, 'dd', 'user_setpoint',
+                                    'units')
+        dd_user = get_status_value(status_info, 'dd', 'position')
+        dd_dial = get_status_value(status_info, 'dd', 'dial_position', 'value')
+
+        yag_zoom_units = get_status_value(status_info, 'yag_zoom',
+                                          'user_setpoint', 'units')
+        yag_zoom_user = get_status_value(status_info, 'yag_zoom', 'position')
+        yag_zoom_dial = get_status_value(status_info, 'yag_zoom',
+                                         'dial_position', 'value')
+
+        def form(left_str, center_str, right_str):
+            return f'{left_str:<15}{center_str:>25}{right_str:>25}'
+
+        def ff(float_str):
```

Comment:
I keep having to format the numbers to fewer decimal points, maybe I should be having an option in the `get_status_value` for that?

Reply:
Given that many values will be floating points, that seems reasonable. However, you might not want `get_status_value` to return a `str` instead a `float`, so maybe a new function `get_status_value_as_float` (or something less verbose) with the `precision` kwarg? 🤷‍♂️

Reply:
solved in #718